x-pack/plugins/saved_objects_tagging/public/management/tag_management_page.tsx

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect, useCallback, useState, useMemo, FC } from 'react';
+import useMount from 'react-use/lib/useMount';
+import { EuiPageContent } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ChromeBreadcrumb, CoreStart } from 'src/core/public';
+import { TagWithRelations, TagsCapabilities } from '../../common';
+import { getCreateModalOpener, getEditModalOpener } from '../components/edition_modal';
+import { ITagInternalClient } from '../tags';
+import { Header, TagTable } from './components';
+import { getTagConnectionsUrl } from './utils';
+
+interface TagManagementPageParams {
+  setBreadcrumbs: (crumbs: ChromeBreadcrumb[]) => void;
+  core: CoreStart;
+  tagClient: ITagInternalClient;
+  capabilities: TagsCapabilities;
+}
+
+export const TagManagementPage: FC<TagManagementPageParams> = ({
+  setBreadcrumbs,
+  core,
+  tagClient,
+  capabilities,
+}) => {
+  const { overlays, notifications, application, http } = core;
+  const [loading, setLoading] = useState<boolean>(false);
+  const [allTags, setAllTags] = useState<TagWithRelations[]>([]);
+  const [selectedTags, setSelectedTags] = useState<TagWithRelations[]>([]);
+
+  const createModalOpener = useMemo(() => getCreateModalOpener({ overlays, tagClient }), [
+    overlays,
+    tagClient,
+  ]);
+  const editModalOpener = useMemo(() => getEditModalOpener({ overlays, tagClient }), [
+    overlays,
+    tagClient,
+  ]);
+
+  useEffect(() => {
+    setBreadcrumbs([
+      {
+        text: i18n.translate('xpack.savedObjectsTagging.management.breadcrumb.index', {
+          defaultMessage: 'Tags',
+        }),
+        href: '/',
+      },
+    ]);
+  }, [setBreadcrumbs]);
+
+  const fetchTags = useCallback(async () => {
+    setLoading(true);
+    const { tags } = await tagClient.find({
```

Comment:
That's a valid remark. However, initially, the table was async-loading the current page instead of all the tags, as it is done for the SO management table. I felt back to 'fetch all' because of some limitations of the EUI table components regarding batch selection. As it may change in the near future, causing the implem to go back to the `find({page: currentPage, perPage: currentPerPage})`, so I would keep it that way at least until we have the 'batch selection' menu implemented.

Reply:
Makes sense to me